### PR TITLE
Move `Defaults to` to end of arg docstring and standardise values

### PR DESCRIPTION
**Pull request by SamuelMarks**
# What does this PR do?

Hey it's that annoying fellow again, the one from https://github.com/keras-team/keras/pull/17748

## Before submitting

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: keras_nlp/layers/f_net_encoder.py [hunk 1]
```diff
@@ -37,18 +37,20 @@ class FNetEncoder(keras.layers.Layer):
...
-        kernel_initializer: "string" or `keras.initializers` initializer,
-            defaults to "glorot_uniform". The kernel initializer for the dense
-            layers.
-        bias_initializer: "string" or `keras.initializers` initializer,
-            defaults to "zeros". The bias initializer for the dense layers.
-        name: string, defaults to None. The name of the layer.
+            Defaults to "relu".
+        layer_norm_epsilon: float. The epsilon value in layer
+            normalization components. Defaults to `1e-5`.
+        kernel_initializer: "string" or `keras.initializers` initializer.
```
Comment (mattdangerw) [inline keras_nlp/layers/f_net_encoder.py:47]: preexisting, but we shouldn't quote "string", here and three lines down

File: keras_nlp/layers/f_net_encoder.py [hunk 2]
```diff
@@ -37,18 +37,20 @@ class FNetEncoder(keras.layers.Layer):
...
+            The kernel initializer for the dense layers.
+            Defaults to "glorot_uniform".
```
Comment (mattdangerw) [inline keras_nlp/layers/f_net_encoder.py:49]: backticks around `"glorot_uniform"`

File: keras_nlp/layers/f_net_encoder.py [hunk 3]
```diff
@@ -37,18 +37,20 @@ class FNetEncoder(keras.layers.Layer):
...
+        bias_initializer: "string" or `keras.initializers` initializer.
+            The bias initializer for the dense layers.
+            Defaults to "zeros".
```
Comment (mattdangerw) [inline keras_nlp/layers/f_net_encoder.py:52]: backticks for `"zeros'`

File: keras_nlp/layers/masked_lm_head.py [hunk 4]
```diff
@@ -56,15 +56,15 @@ class MaskedLMHead(keras.layers.Layer):
...
-            the dense and multiheaded attention layers.
-        bias_initializer: string or `keras.initializers` initializer,
-            defaults to "zeros". The bias initializer for
-            the dense and multiheaded attention layers.
-        name: string, defaults to None. The name of the layer.
+        layer_norm_epsilon: float. The epsilon value in layer
+            normalization components. Defaults to `1e-5`.
+        kernel_initializer: string or `keras.initializers` initializer.
+            The kernel initializer for the dense and multiheaded
+            attention layers. Defaults to "glorot_uniform".
```
Comment (mattdangerw) [inline keras_nlp/layers/masked_lm_head.py:63]: backticks for "glorot_uniform"

File: keras_nlp/layers/masked_lm_head.py [hunk 5]
```diff
@@ -56,15 +56,15 @@ class MaskedLMHead(keras.layers.Layer):
...
+        bias_initializer: string or `keras.initializers` initializer.
+            The bias initializer for the dense and multiheaded
+            attention layers. Defaults to "zeros".
```
Comment (mattdangerw) [inline keras_nlp/layers/masked_lm_head.py:66]: backticks for "zeros"

File: keras_nlp/layers/masked_lm_mask_generator.py [hunk 6]
```diff
@@ -42,22 +42,24 @@ class MaskedLMMaskGenerator(keras.layers.Layer):
...
+        mask_selection_length: int. Maximum number of tokens
             selected for  masking in each sequence. If set, the output
             `mask_positions`, `mask_ids` and `mask_weights` will be padded
             to dense tensors of length `mask_selection_length`,
             otherwise the output will be a RaggedTensor.
-        unselectable_token_ids: A list of tokens, defaults to [0] (the default
-            `padding_token_id`).
-        mask_token_rate: float, defaults to 0.8. `mask_token_rate` must be
+            Defaults to `None`.
+        unselectable_token_ids: A list of tokens, [0] is default
```
Comment (mattdangerw) [inline keras_nlp/layers/masked_lm_mask_generator.py:51]: remove this first reference to default

File: keras_nlp/layers/transformer_decoder.py [hunk 7]
```diff
@@ -57,23 +57,25 @@ class TransformerDecoder(keras.layers.Layer):
...
-        layer_norm_epsilon: float, defaults to 1e-5. The eps value in layer
-            normalization components.
-        kernel_initializer: string or `keras.initializers` initializer,
-            defaults to "glorot_uniform". The kernel initializer for
-            the dense and multiheaded attention layers.
-        bias_initializer: string or `keras.initializers` initializer,
-            defaults to "zeros". The bias initializer for
-            the dense and multiheaded attention layers.
-        normalize_first: bool. Defaults to False. If True, the inputs to the
+            Defaults to "relu".
```
Comment (mattdangerw) [inline keras_nlp/layers/transformer_decoder.py:64]: backticks, and so on for any string literal defaults across the rest of the PR.

File: keras_nlp/tokenizers/word_piece_tokenizer_trainer.py [hunk 8]
```diff
@@ -51,25 +51,26 @@ def compute_word_piece_vocabulary(
...
-        strip_accents: bool, defaults to `False`. If true, all accent marks will
-            be removed from text before tokenization.
-        split: bool, defaults to `True`. If true, input will be split on
+        vocabulary_output_file: str. The location to write a
+            vocabulary file. defaults to `None`.
+        lowercase: bool. If `True`, the input text will be
+            lowercased before tokenization. Defaults to `False`.
+        strip_accents: bool. If true, all accent marks will
+            be removed from text before tokenization. Defaults to `False`.
+        split: bool. If True, input will be split on
```
Comment (mattdangerw) [inline keras_nlp/tokenizers/word_piece_tokenizer_trainer.py:60]: backticks for True/False consistently in this file.

**Review by mattdangerw — APPROVED** [review]
Thanks! This is great. Dropped a few minor comments.

File: keras_nlp/layers/masked_lm_mask_generator.py (hunk 6 above)
Comment (SamuelMarks) [reply keras_nlp/layers/masked_lm_mask_generator.py:50]: Not sure what language you want used here

Comment (mattdangerw) [reply keras_nlp/layers/masked_lm_mask_generator.py:50]: I guess I would do this ->

unselectable_token_ids: A list of tokens id that should not be considered eligible for masking. Defaults to `[0]`, where 0 is assumed to be the padding token id.

Would that run counter to what you are trying to do?

Comment (SamuelMarks) [reply keras_nlp/layers/masked_lm_mask_generator.py:50]: Yeah because its ambiguous whether the default value becomes
> `[0]`, where 0 is assumed to be the padding token id

Or
> `[0]`

PS: Maybe some kind of `itemgetter(0)` would make more sense as the default here? - Just trying to think of this in SQL, CLI, and SDK contexts.

Comment (mattdangerw) [reply keras_nlp/layers/masked_lm_mask_generator.py:50]: I'm not sure how `itemgetter(0)` would work. This isn't an argument we use to index into something, this is a set of values we use to compare against similarly tokenized input in this same integer space.
 
Maybe just say

```
unselectable_token_ids: A list of tokens id that should not be considered eligible for masking. By default we assume `0` corresponds to a padding token and ignore it. Defaults to `[0]`.
```

A little redundant but keeps the consistent form on the last sentence.

Comment (SamuelMarks) [reply keras_nlp/layers/masked_lm_mask_generator.py]: Done

**Review by mattdangerw — APPROVED** [review]
Thanks!
